scripts/package.ts

```diff
@@ -106,6 +107,68 @@ function getVersionSuffix(feature: string, debug: boolean): string {
     return `${debugSuffix}${featureSuffix}${commitSuffix}`
 }
 
+/**
+ * @returns true if curl is available
+ */
+function isCurlAvailable(): boolean {
+    try {
+        child_process.execFileSync('curl', ['--version']);
+        return true;
+    } catch {
+        return false;
+    }
+}
+
+/**
+ * Small utility to download files.
+ */
+function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
+    if (platform() !== 'linux') {
+        return;
+    }
+
+    if (!isCurlAvailable()) {
+        return;
+    }
+
+    // Create output directory if it doesn't exist
+    if (!nodefs.existsSync(outputDir)) {
+        nodefs.mkdirSync(outputDir, { recursive: true });
+    }
+
+    urls.forEach(url => {
+        const filePath = path.join(outputDir, outputFile || '');
+
```

Comment:
Security Concern: The `downloadFiles` function is downloading files from a hardcoded GitHub repository URL[^1]. This could potentially lead to security risks if the repository is compromised or if an attacker gains control of the specified tag.

[^1]: While not a direct CWE, this relates to CWE-829: Inclusion of Functionality from Untrusted Control Sphere - https://cwe.mitre.org/data/definitions/829.html
```suggestion
// Consider using a more secure method for retrieving files
// For example, use a verified package manager or a secure, internal file server
function downloadFiles(urls: string[], outputDir: string, outputFile: string): void {
    // Implement a secure file retrieval method here
    // For example, use a package manager or a secure, internal file server
    console.log('Implement secure file retrieval here');
}
```